incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,58 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
+  tag: "latest"
```

Comment:
I have had latest tags take down production ;( So I kinda agree with @prydonius on this